golang/sqlboiler/templates/00_struct.tpl

```diff
@@ -143,3 +144,51 @@ func (o {{$modelName}}) {{titleCase $column.Name}}() string {
 }
 {{- end -}}
 {{- end }}
+
+{{/* Add resolvers to FK relationships */}}
+{{range .Table.FKeys -}}
+{{- $txt := txtsFromFKey $dot.Tables $dot.Table . -}}
+// {{$txt.ForeignTable.NameGo}} pointed to by the foreign key
+func (o {{$modelName}}) {{$txt.ForeignTable.NameGo}}() *{{$txt.ForeignTable.NameGo}} {
+  if o.model.R == nil || o.model.R.{{$txt.Function.Name}} == nil {
+    return nil
+  }
+  return &{{$txt.ForeignTable.NameGo}}{
+		model: *o.model.R.{{$txt.Function.Name}},
+		db:    o.db,
+	}
+}
+{{end -}}
+
+{{/* Add resolvers to one relationships */}}
+{{range .Table.ToOneRelationships -}}
+{{- $txt := txtsFromOneToOne $dot.Tables $dot.Table . -}}
+// {{$txt.ForeignTable.NameGo}} pointed to by the foreign key
+func (o {{$modelName}}) {{$txt.ForeignTable.NameGo}}() *{{$txt.ForeignTable.NameGo}} {
+  if o.model.R == nil || o.model.R.{{$txt.Function.Name}} == nil {
+    return nil
+  }
+  return &{{$txt.ForeignTable.NameGo}}{
+		model: *o.model.R.{{$txt.Function.Name}},
+		db:    o.db,
```

Comment:
Template variables used for foreign table struct name and _R_ field are based on how the default template folder is generating. Please see: 
https://github.com/volatiletech/sqlboiler/blob/7e0eca627f995e2476d637ada0c3b70ecb5d837c/templates/00_struct.tpl#L41